util/stmtsummary/statement_summary.go

```diff
@@ -364,6 +370,7 @@ func (ssMap *stmtSummaryByDigestMap) ToHistoryDatum(user *auth.UserIdentity, isS
 		records := value.(*stmtSummaryByDigest).toHistoryDatum(historySize, user, isSuper)
 		rows = append(rows, records...)
 	}
+	rows = append(rows, ssMap.other.toHistoryDatum(historySize)...)
```

Comment:
Ditto.

Reply:
@djshow832 done.
but I also moved the line getting historySize ahead, please aware.